src/main/resources/assets/tconstruct/book/materials_and_you/en_us/modifiers/defense.json

```diff
@@ -7,6 +7,10 @@
     {
       "text": "Like many other modifier types, defense modifiers require defense slots to be applied. Typically, armor with a large number of defense slots start with fewer upgrade slots, meaning functionality is sacrificed for protection.",
       "paragraph": true
+    },
+	 {
+      "text": "Abilities are beyond my knowledge but can be found in Mighty smelting, along with Defense modifiers.",
```

Comment:
Let me clarify exactly what this should say. There are four pages in Materials and You:
* Upgrades: tells you to look in encyclopedia/puny
* Abilities: tells you to look in encyclopedia/mighty
* Slotless: tells you to look in encyclopedia/puny
* Defense: tells you to look in encyclopedia, and eventually mighty

I want you to change defense to tell you defense modifiers are in encyclopedia and mighty smelting. You don't need to mention any other modifier types, the other pages cover those.